client/src/components/photo_feed.jsx

```diff
@@ -1,32 +1,43 @@
 import React from 'react';
+import {FeedStyle} from '../styles/Modal.js';
+import {TopModal} from './modal/mosaic_top_modal.jsx';
```

Comment:
Are you importing top modal with the {} curly brackets because of how you exported it? Or is there a specific reason you're using the {}?